presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Hadoop.java

```diff
@@ -31,7 +31,9 @@
 public final class Hadoop
         implements EnvironmentExtender
 {
-    public static final String CONTAINER_PRESTO_HIVE_PROPERTIES = "/docker/presto-product-tests/conf/presto/etc/catalog/hive.properties";
+    public static final String CONTAINER_PRESTO_CATALOG = "/docker/presto-product-tests/conf/presto/etc/catalog/";
+    public static final String CONTAINER_PRESTO_HIVE_PROPERTIES = CONTAINER_PRESTO_CATALOG + "hive.properties";
+    public static final String CONTAINER_PRESTO_KAFKA_PROPERTIES = CONTAINER_PRESTO_CATALOG + "kafka.properties";
```

Comment:
This constant doesn't seem to belong here, as `Hadoop` class doesn't have anything to do with `kafka.properties`.

We could define `/docker/presto-product-tests/conf/presto` as a const in `Standard`